humanapi/connect-client
# COR-3268: Add error event

## What did you implement
- Add error event and message when container is not found
- Bump to 2.1.0

## How can we verify it
- Include https://cdn.humanapi.co/humanapi-connect-client@2.1.0-beta.1.js in your code
- In your connect integration don't add a `.hapi__token-container` element, an error should pop up in the console
- You can also listen to the errors by adding an event listener:
  ```javascript
  HumanConnect.on("error", response => console.error("error", response));
  ```
